ui/src/app/components/rename-modal.spec.tsx

```diff
@@ -34,6 +34,9 @@ describe('should show error if new name already exist', () => {
     ['123.ipynb', '123.ipynb'],
     ['123.Rmd', '123'],
     ['123.Rmd', '123.Rmd'],
+    ['123.Rmd', '123.rMD'],
```

Comment:
I'm not sure that this is the behavior that we want.

I'm imagining a user who made a typo and now wants to fix it.

Reply:
good point...I'll fix this